firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -4703,6 +4703,11 @@ extension String {
         tableName: nil,
         value: "Add to Reading List",
         comment: "Accessibility label for action adding current page to reading list.")
+    public static let TabLocationReloadButtonMoreOptionsAccessibilityLabel = MZLocalizedString(
+        key: "Address.Bar.MoreOptions",
+        tableName: nil,
+        value: "More Options",
+        comment: "Accessibility label for action showing more options")
```

Comment:
@thatswinnie @adudenamedruby do you know if for such items you ever talked with accessibility team or anyone for adding values like these?

Reply:
As far as I know, copy should be providing strings for strings for accessibility.

Also, @imsanchit, please take a look at the wiki for adding strings. New strings will require a specific format. :)

Reply:
@nbhasin2 @adudenamedruby Please note these strings aren't suggested by anyone, I have added them as per my understanding as a part of this fix, we can change these values as per your suggestions. Also, I have updated the PR with the required format, please have a look and feel free to point out any required corrections.